src/dmd/chkformat.d

```diff
@@ -915,6 +940,26 @@ Format parseGenericFormatSpecifier(scope const char[] format,
             break;
     }
 
+    version (CRuntime_Glibc)
```

Comment:
This depends on the host compiler - not the currently targeted `libc`

Reply:
Is there an easy way to enable this test only for the glibc runtime ?

Reply:
No, it's not. DMD can use the GNU C library, just like GDC and LDC.

Reply:
Not sure what you are talking about, `version (CRuntime_Glibc)` is evaluated when *compiling dmd itself* (regardless of the actual host compiler).
But these extensions are only available when the *user code* is linked to glibc.

Reply:
You're right 🤦🏼